challenges/2022-06-14-two-sum-numbers/solutions/javascript/Moses-Alero/TwosumNumbers.js

```diff
@@ -1,9 +1,8 @@
 const TwoSumNumbers = function (number, numbersArray){
 
-   for(let i = 0; i < numbersArray.length;i++){
+   while(numbersArray.length > 0){
         const summation = number - numbersArray[0];
```

Comment:
`summation` isn't really an appropriate variable name. This is really an `addend` - a number added to `numbersArray[0]` to get `number`. In this case, `number` is the sum.

Reply:
So what you're asking me to do is change it such that it becomes
 `number = addend - numbersArray[0]` right?

Reply:
Oops! I see that the dash I used in the first sentence made it seem like I was talking about subtraction. Sorry that was confusing. I meant to suggest `const addend = number - numbersArray[0]' and keep `number` as your input. `difference` could also be used instead of `addend`

Reply:
I think I'd use `difference` instead.
I believe that's more relatable and easier to understand unlike addend

Reply:
I've resolved that issue you can check and see if there's anything else